scalability_and_performance/compute-resource-quotas.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="compute-resource-quotas_{context}"]
+=  Using Quotas and Limit Ranges
+include::_attributes/common-attributes.adoc[]
+:context: using-quotas-and-limit-ranges
+
+A resource quota, defined by a `*ResourceQuota*` object, provides constraints that limit aggregate resource consumption per project. It can limit the quantity of objects that can be created in a project by type, as well as the total amount of compute resources and storage that may be consumed by resources in that project.
```

Comment:
```suggestion
:context: using-quotas-and-limit-ranges

toc::[]

A resource quota, defined by a `*ResourceQuota*` object, provides constraints that limit aggregate resource consumption per project. It can limit the quantity of objects that can be created in a project by type, as well as the total amount of compute resources and storage that may be consumed by resources in that project.
```

Reply:
This is done. Thanks @bscott-rh